js/src/util/backdrop.js

```diff
@@ -116,7 +116,11 @@ class Backdrop {
 
     EventHandler.off(this._element, EVENT_MOUSEDOWN)
 
-    this._getElement().parentNode.removeChild(this._element)
+    const { parentNode } = this._getElement()
```

Comment:
```suggestion
    const { parentNode } = this._element
```

above of it, seems that it is definded

Reply:
You're right - using `this._element` would be consistent with what's above.

However, I made this change... then reversed it. The reason is that - before this PR - if someone calls `dispose()` when `this._element` is null, there is no error (`EventHandler.off()` has a check for passing in a null 1st argument. But if we DID include this change, then you get this:

> Cannot destructure property 'parentNode' of this.'_element' as it is null

But please tell me if you disagree :)

Reply:
I saw the code. 
You 've done the right approach. 

However the rest code needs to be re-visited